_clanky/neziskovky-automatizace-dashboardu.md

```diff
@@ -0,0 +1,97 @@
+
+* účet Google - účet Google jistě znáte a mnozí z vás ho už i máte, protože existuje široká paleta nástrojů které je možné s tímto účtem využívat jako například YouTube nebo Gmail. Pro naše účely je tento účet důležitý proto že nám umožní zdarma využívat také služby Google Data Studio pro vytvoření dashboardu a developerskou konzoli pro komunikaci skriptu s Google sheets.
+
+##Editor kódu
+
+První ingrediencí, kterou budeme potřebovat, je textový editor kódu, ve kterém budeme spouštět předpřipravený skript pro aktualizaci dat. Můžete použít kterýkoliv z volně dostupných editorů, jako například [Notepad++][NOTEPAD], [PyCharm][PYCHARM] nebo [Atom][ATOM]. Všechny jsou zdarma a fungují na na všech platformách - Linux, OS X a také Windows. Po stažení a instalaci jste připraveni kterýkoliv z nich začít hned používat. My jsme využili editor [Sublime][SUBLIME], ale na tom, který si zvolíte, vůbec nezáleží.
+
+##Python
+
+K tomu, aby váš textový editor dokázal interpretovat, co do něj píšete, bude nutné si nainstalovat programovací jazyk [Python][PYTHON]. Ten si [stáhnete z úvodní obrazovky][PYTHON] a následně, po stažení, jej opět nainstalujete obdobně jako textový editor. Pak bude nutné přidat ještě knihovny pro rozpoznání metod a příkazů. Knihovny nainstalujete zapsáním příkazu do [příkazového řádku][RADEK], kde napíšete - **“pip install pandas”** a stisknete enter. Postup opakujte ještě dvakrát: pro příkazy **“pip install gspread”** a **“pip install gspread_dataframe”**. Po doběhnutí poslední instalace jste co se týče Pythonu připraveni.
+
+##Účet Google
+
+[Účet google][GOOGLE] si vytvoříte velice jednoduše a rychle. S velkou pravděpodobností takovýto účet už stejně máte. S účtem Google získáte přístup do mnoha nástrojů, avšak pro naše účely bude stačit [Google Drive][DRIVE], [Google Sheets][SHEETS], [Google Data Studio][DATASTUDIO]. 
+
+Po aktivaci účtu google je potřeba zajít na stránku [console.developers.google.com][CONSOLE], kde si vytvoříte nový projekt. Zde je nutné napsat **pouze název projektu** a pak stisknout tlačítko Vytvořit/Create. Poté bude nutné povolit také **Google Drive API** a **Google Sheets API**. Abyste tyto nástroje mohli povolit, je nutné být v úvodní obrazovce projektu, který jste právě vytvořili a ve vyhledávací liště zadat jak Google Drive API, tak i Google Sheets API a povolit každou zvlášť, a to kliknutím na tlačítko **Povolit/Enable**. API slouží k vzájemné komunikaci mezi skriptem, google sheets a google drive.
```

Comment:
Vytovřit/Create je obyčejně, a povolit je tučně. To je nekonzistntní, a také není jasné, co to tučně znamená. Je to příklad toho zbytečného tučnění. Pokud chceš ukázat kód, tak `takto`.